db/migrate/20200523170314_add_fk_rubygem_id_nullify_constraint_to_dependencies.rb

```diff
@@ -0,0 +1,5 @@
+class AddFkRubygemIdNullifyConstraintToDependencies < ActiveRecord::Migration[6.0]
+  def change
+    add_foreign_key :dependencies, :rubygems, on_delete: :nullify
```

Comment:
Why nullify?

Reply:
https://github.com/rubygems/rubygems.org/pull/2361 
according to this pr and its thread + hooks on model later, we set id to null when gem got yanked